src/main/java/com/linecorp/armeria/server/http/tomcat/TomcatService.java

```diff
@@ -73,6 +76,10 @@ public static TomcatService forFileSystem(Path docBase) {
         return TomcatServiceBuilder.forFileSystem(docBase).build();
     }
 
+    public static HttpService forConnector(String hostname, Connector connector) {
```

Comment:
But Connector contains reference to other variables. so it can be inferred.

`connector().getService().getServer().getCatalina()`